box.py

```diff
@@ -460,7 +463,8 @@ def __box_config(self):
         return out
 
     def __convert_and_store(self, item, value):
-        if item in self._box_config['__converted']:
+        if item in self._box_config['__converted'] or
```

Comment:
This needs `()` around it to span multiple lines. (Is what TravisCI is complaining).  Not a blocking for merging into dev, just clearing up why that was complaining.